merge-scripts/goerlishadow-4.vars

```diff
@@ -1,3 +1,15 @@
+# Uncomment the following if you want to start validator with deposit cli keystores
```

Comment:
Maybe have the possibility to choose this via a command line argument? I feel having to uncomment/comment is a bit clunky and might not give the best UX

Reply:
umm... the issue will be people forgetting to add the flag between runs (highly likely, happens all the time), could lead to messed up imports.

or we can have an interactive input to pick this data for first time (and the script will persist the choice and options next time around in a file). This can also be asked for the jwt-secret, i.e. the personalization options which are a bit different from configuration options. so for the @philknows 's guide, one wouldn't need to open the vars file at all.

(for which the person will have to edit vars assuming they know what they are doing)

Reply:
Sounds good to me. As long as the cost of implementation is not more than the benefit...